app/services/wildcard_url_checker.rb

```diff
@@ -0,0 +1,12 @@
+module WildcardUrlChecker
+
+  def self.check_url(url, url_to_check)
+    puts "url: " + url
```

Comment:
Those `puts` should probably be removed ;)

Reply:
Ops! I forgot some debug code.